task1/src/main/java/com/fundacionjala/movies/RegularMovie.java

```diff
@@ -0,0 +1,22 @@
+package com.fundacionjala.movies;
+
+/**
+ * Created by Mijhail on 11/06/2016.
+ */
+public class RegularMovie extends Movie {
+
+    public RegularMovie(String title) {
+        super(title);
+    }
+
+    @Override
+    public double calculateChargeMovie(int daysRented) {
+        int charge = 2;
```

Comment:
If you notice you are repeating the variables charge, days, and benefit in the RegularMovie, NewReleaseMovie, and ChildrenMovie classes.
You can abstract those variable in the Movie class and inherent them